routes/web.php

```
@@ -19,6 +20,12 @@
 
 Route::middleware('auth')->prefix('/app')->group(function () {
     Route::get('/', AppController::class)->name('app');
+    
+    Route::get('/parties', [PartyController::class, 'index'])->name('parties.index');
+    Route::get('/parties/create', [PartyController::class, 'create'])->name('parties.create');//->can('create', Party::class); // @TODO can't figure out why unauthorized
```

@imacrayon This one throws me.  Is my policy wrong or something else?

---

@TODO can't figure out why unauthorized

---

Make sure you have `use App\Models\Party` at the top of the file. It’s fails silently if you don’t have the class imported.

---

Doh... I didn't have `use App\Models\Party;`.